src/LearnerData/SFA.DAS.LearnerData/Application/UpdateLearner/EarningsApiClientExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.Extensions.Logging;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests.LearnerData;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.LearnerData.Application.UpdateLearner;
+
+/// <summary>
+/// Extensions for the Earnings API client to handle updates related to learners. This is specifically for use with the UpdateLearnerCommandHandler.
+/// </summary>
+internal static class EarningsApiClientExtensions
```

Comment:
I like this approach for handling the earnings api calls a lot. Passing the command and letting this class work out what to pass to the earnings endpoints is clean, but it also means if we switch it up to a single endpoint in future this will be the only class that has to change.